capellambse/svg/decorations.py

```diff
@@ -17,18 +17,22 @@
 """Default icon padding (left/right side)."""
 feature_space = 24
 """Default margins/padding (top/bot and left/right) for feature text."""
+text_margin = 5
+"""Default margins/padding (left and right) for label text."""
 
 function_ports = {"FIP", "FOP"}
 directed_component_ports = {"CP_IN", "CP_OUT"}
 component_ports = directed_component_ports | {"PP", "CP_INOUT", "CP_UNSET"}
 all_ports = function_ports | component_ports
 all_directed_ports = directed_component_ports | function_ports
-start_aligned = {
+start_anchored = {
     "LogicalComponent",
     "LogicalActor",
     "LogicalHumanActor",
     "LogicalHumanComponent",
+    "ModeRegion",
 }
+left_aligned = {"ModeRegion"}
```

Comment:
Can we change the text alignment setting to be a property in `capstyle.STYLES`?

Reply:
We moved all text based decoration rules into `diagram.capstyle.STYLES`. Please have a look again. And tell me if you like this or not.